geomet/tests/wkt_test.py

```diff
@@ -1227,7 +1227,8 @@ def test(self):
         test_cases = [
             [(-1.000000000000000, 16), '-1.' + '0' * 16],
             [(-83.2496395, 16), '-83.2496395000000000'],
-            [(35.917330500000006, 16), '35.9173305000000060']
+            [(35.917330500000006, 16), '35.9173305000000060'],
+            [(6e-6, 16), '0.0000060000000000']
```

Comment:
I tested this locally with yet another test case:

```
[(1e16, 3), '10000000000000000.000'],
```

This breaks:

```
ERROR: test (geomet.tests.wkt_test.TestRoundAndPad)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/path/to/proj/geomet/geomet/tests/wkt_test.py", line 1237, in test
    self.assertEqual(expected, wkt._round_and_pad(*args))
  File "/path/to/proj/geomet/geomet/wkt.py", line 203, in _round_and_pad
    rounded += '0' * (decimals - len(rounded.split('.')[1]))
IndexError: list index out of range
```

I have a suggestion for a fix below.

Reply:
I would advise add the at least the following test cases:

```
            [(1e16, 3), '10000000000000000.000'],
            [(3.14e16, 3), '31400000000000000.000'],
            [(3.14e-16, 18), '0.000000000000000314'],
```

@tomplex Can think of any others cases that would be relevant here?